packages/ui/react-ui-form/src/components/Form/FormContent.tsx

```diff
@@ -71,7 +67,28 @@ export const FormField = ({ property, path, readonly, inline, lookupComponent, C
   }
 
   const jsonPath = createJsonPath(path ?? []);
-  const InputComponent = Custom?.[jsonPath] || getInputComponent(type, format);
+  const CustomComponent = Custom?.[jsonPath];
+  if (CustomComponent) {
+    return (
+      <div>
```

Comment:
why's this div needed here?

Reply:
+1

Reply:
So that any children get treated as a single item in the parent flex layout.

Reply:
my gut says that should be up to the custom component unless there's some reason it can't be

Reply:
Left a TODO. It's not really the core of this PR.